lib/bundle/extend/os/linux/skipper.rb

```diff
@@ -3,8 +3,18 @@
 module Bundle
   module Skipper
     class << self
+      def macos_only_entry?(entry)
+        [:cask, :mas].include?(entry.type)
+      end
+
+      def macos_only_tap?(entry)
+        entry.type == :tap && entry.name == "homebrew/cask"
+      end
+
       def skip?(entry, silent: false)
-        return generic_skip?(entry) unless [:cask, :mas].include?(entry.type)
+        unless macos_only_entry?(entry) || macos_only_tap?(entry)
```

Comment:
`unless ... |||` is pretty hard to follow/parse. Could this be written as an `if` (or two)?